qutip/__init__.py

```diff
@@ -261,10 +261,10 @@
 from qutip.about import *
 
 # Setup pyximport 
-import pyximport
-os.environ['CFLAGS'] = '-O2 -w -ffast-math'
-pyximport.install(setup_args={'include_dirs': [numpy.get_include()]})
-del pyximport
+import qutip.cy.pyxbuilder as pbldr
+os.environ['CFLAGS'] = '-O1 -w -ffast-math'
```

Comment:
Just a question: why do we have a reduced set of flags here compared to `_compiler_flags` in cy/setup.py?